solr/schema.xml

```diff
@@ -160,10 +160,18 @@ attribute with the form `ckan-X.Y` -->
         <field name="provenance" type="string" indexed="true" stored="true" multiValued="false" />
         <field name="spatial_uri" type="string" indexed="true" stored="true" multiValued="false" />
         <field name="contact_uri" type="string" indexed="true" stored="true" multiValued="false" />
-        <field name="contact_point" type="string" indexed="true" stored="true" multiValued="true" />
+        <field name="contact_name" type="text_general" indexed="true" stored="true" />
+        <field name="creator" type="string" indexed="true" stored="true" multiValued="true" />
+        <field name="creator_name" type="text_general" indexed="true" stored="true" />
+        <field name="creator_identifier" type="string" indexed="true" stored="true" />
+
         <field name="resources_formats" type="string" indexed="true" stored="true"
             multiValued="true" />
         <field name="publisher_name" type="string" indexed="true" stored="true" multiValued="false" />
+        <field name="publisher_uri" type="string" indexed="true" stored="true" />
+        <field name="publisher_email" type="text_general" indexed="true" stored="true" />
```

Comment:
@hcvdwerf what is the difference between `text_general` and `string` here? I thought it should be all string.

Reply:
Use string for fields where exact matching is required and no text analysis should be performed.
Use text_general for fields where you need full-text search capabilities, tokenization, and case insensitivity.]

Reply:
I think uri is exact and e-mail maybe search partly possible. I copy paste it honestly

Reply:
fixt